var/build-config.json

```diff
@@ -1,9 +1,25 @@
 {
-  "boardingLocationTags": ["gtfs:stop_id", "ref"],
+  "boardingLocationTags": [
+    "gtfs:stop_id",
+    "ref"
+  ],
+  "dataImportReport": true,
   "embedRouterConfig": false,
   "osmDefaults": {
     "timeZone": "America/New_York"
   },
+  "transitFeeds": [
+    {
+      "type": "gtfs",
+      "feedId": "massport-ma-us",
+      "source": "https://data.trilliumtransit.com/gtfs/massport-ma-us/massport-ma-us.zip"
```

Comment:
question: why use HTTP for this, but not for the main GTFS?

Reply:
The main GTFS destination can change based on an environment variable. I thought referencing environment variables wouldn't work here but it turns out it works! I've since created consistency by defining all the external resources in this config! ✨